letsencrypt-auto-source/letsencrypt-auto

```diff
@@ -500,6 +500,7 @@ Bootstrap() {
     echo "You will need to bootstrap, configure virtualenv, and run pip install manually."
     echo "Please see https://letsencrypt.readthedocs.org/en/latest/contributing.html#prerequisites"
```

Comment:
I like that approach in theory, but I'm not sure how well it works in practice. I haven't seen a lot of instances of it, but the comment that caused me to create this PR can be found [here](https://github.com/certbot/certbot/issues/2540#issue-136091238). The cause of the problem is not very clear.